MotorControl/MotorControl/Resources/Finger_Tapping.json

```diff
@@ -8,7 +8,7 @@
                 "identifier":"introduction",
                 "type":"overview",
                 "title":"Finger Tapping",
-                "text":"In this activity we will measure your finger tapping speed with each hand; this may reflect severity of motor symptoms in Parkinson’s Disease.",
+                "text":"In this activity we will measure your finger tapping speed with each hand.",
```

Comment:
"In this activity, we will measure your finger tapping speed and accuracy with each hand"

Reply:
Thanks